shared/attributes.asciidoc

```diff
@@ -123,13 +123,19 @@ ifdef::elasticsearch-root[]
 endif::elasticsearch-root[]
 
 //////////
-Kibana app names
+Kibana app and UI names
 //////////
 
-:uptime-app: Uptime app
-:logs-app: Logs app
-:metrics-app: Metrics app
-:siem-app: SIEM app
+:uptime-app:         Uptime app
+:logs-app:           Logs app
+:metrics-app:        Metrics app
+:siem-app:           SIEM app
+:ml-app:             *Machine Learning*
```

Comment:
It's possible I'm doing something wrong, but I'm not seeing the bold formatting coming through when using these attributes. Is that what you were trying to do?

Reply:
You're right, it doesn't work. I've removed the asterisks and can add them as necessary in the source files.